app/assets/stylesheets/application/components/checkin.scss

```diff
@@ -0,0 +1,67 @@
+.checkin {
+  .media {
+    .media-body {
+      width: 100%;
+
+      .top {
+        height: 17px;
+        line-height: 17px;
+
+        .name {
```

Comment:
Nesting should be no greater than 4, but was 5<br>Selector should have depth of applicability no greater than 2, but was 5